src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,50 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> getListOfNumberWhereSumDigPowEqualsNumber(long startValueOfRange, long endValueOfRange) {
```

Comment:
Gut: Name angepasst, aber immer noch nicht so aussagekräftig. z.B. überflüssig: getListOf nicht
Und Test Compile Errors, meint zumindest der Compiler

Reply:
gute Namenswahl (evtl. "value2 nicht erforderlich) => for-schleife wird auch sehr lang